### PR TITLE
feat: added contact us form

### PR DESCRIPTION
**Pull request by suyogkokaje**
Please have a look.

### CODE REVIEW

**Review by nandini7637 — CHANGES_REQUESTED** [review]
Add class to body while adding css, people might have problem using it in future

**Review by nandini7637 — APPROVED** [review]

**Review by Abirpal202049 — APPROVED** [review]

**Review by saikatmu31 — APPROVED** [review]

File: Forms/Contact Us Form_01/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Music Player
+<h3> Tech Stack Used <img src = "https://media2.giphy.com/media/QssGEmpkyEOhBCb7e1/giphy.gif?cid=ecf05e47a0n3gi1bfqntqmob8g9aid1oyj2wr3ds3mg700bl&rid=giphy.gif" width = 32px> </h3>
+<p align="left"> <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/> <a href="https://www.w3schools.com/css/" target="_blank"><img src="https://img.icons8.com/color/48/000000/css3.png"/></a>   
+</p>
+<hr>
+
+### Preview
+![](./assets/preview.gif)
```
Comment (thepranaygupta) [inline Forms/Contact Us Form_01/README.md:8]: please embed the preview gif directly by drag and drop

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

**Comment by nandini7637** [comment]
@suyogkokaje have you worked on changes?

**Comment by suyogkokaje** [comment]
@thepranaygupta please have a look.

**Review by bhavesh-chaudhari — APPROVED** [review]

**Review by Ovenoboyo — APPROVED** [review]

**Pull request merged**